src/pages/CropSelector/CropTable/CropTableCard.js

```diff
@@ -23,7 +23,16 @@ const CropTableCard = ({ crop, indexKey, showGrowthWindow }) => {
     <>
       {selectedGoalsRedux.length > 0
         && selectedGoalsRedux.map((goal, index) => (
-          <TableCell size="small" style={{ textAlign: 'center' }} key={index} className="goalCells">
+
+          <TableCell
+            size="small"
+            style={{
+              textAlign: 'center',
+              backgroundColor: selectedCropIdsRedux.includes(crop.id) && '#EAEAEA',
```

Comment:
same here, the color has been set in `<TableRow/>` in `RenderTableItems.js`

Reply:
good catch i fully expected to have errors in doing the whole row so i approached it as a per cell issue.